### PR TITLE
Debug. Logging modeChoice alternatives in modeChoiceMultinomialLogit

### PR DESCRIPTION
**Pull request by nikolayilyin**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2723)

### CODE REVIEW

**Comment by nikolayilyin** [comment]
#2716

**Comment by nikolayilyin** [comment]
test!

File: src/main/scala/beam/agentsim/events/ModeChoiceOccurredEvent.scala [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    personId: String,
+    alternatives: IndexedSeq[EmbodiedBeamTrip],
+    modeCostTimeTransfers: Map[String, AltCostTimeTransfer],
+    alternativesUtility: Map[String, AltUtility],
+    chosenAlternativeIdx: Int
+  ): ModeChoiceOccurredEvent = {
+
+    // just to have an approximate event time.
+    // not used in calculations at all.
+    val time = alternatives.head.legs.head.beamLeg.startTime
```
[the remark this answers is not in the document]
Comment (REASY) [reply src/main/scala/beam/agentsim/events/ModeChoiceOccurredEvent.scala]: @nikolayilyin as per discussion `alternatives.head.legs.head.beamLeg.startTime` is the right time to take

**Comment by nikolayilyin** [comment]
test!

**Comment by nikolayilyin** [comment]
test!

**Pull request merged**